lib/schemas/AuthenticationSchema.js

```diff
@@ -25,28 +39,36 @@ module.exports = makeSchema({
         'custom',
         'digest',
         'oauth2',
-        'session'
-      ]
+        'session',
+      ],
     },
     test: {
       description: 'A function or request that confirms the authentication is working.',
       oneOf: [
         {$ref: RequestSchema.id},
-        {$ref: FunctionSchema.id}
+        {$ref: FunctionSchema.id},
       ]
     },
     fields: {
       description: 'Fields you can request from the user before they connect your app to Zapier.',
-      $ref: FieldsSchema.id // TODO: can this be DynamicFieldsSchema?
+      $ref: FieldsSchema.id,
+    },
+    connectionLabel: {
```

Comment:
Do we have a concept in the existing docs (dev side or product side), or something already in the CLI that we use besides `connection`? The name itself is fine, and communicates what this does, but `connection` is a new term I think for the CLI. In the UI, we have "connected accounts" and on the platform side I think we've always just said "auth"?

If this was an intentional decision, or there isn't a term, then :+1: